examples/src/createReactAgent/accessExternalContext.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Access External Context in Message Generation
```

Comment:
put more concisely:

context can contain runtime parameters, or connections to dbs, etc

this can affect the system prompt